.pre-commit-config.yaml

```diff
@@ -38,3 +46,12 @@ repos:
       - id: typos
         args: ['--config=typos-config.toml']
         pass_filenames: false
+  - repo: https://github.com/Yelp/detect-secrets
+    rev: v1.5.0
+    hooks:
+      - id: detect-secrets
+        args: ['--exclude-files', 'Tests/*']
```

Comment:
do we really have secrets that we need this pre-commit hook? It's an open source project 🙂

Reply:
It is actually for prevent adding secret keys. Since it is an open source project there should be no secret key, so it should pass. Otherwise (if someone tried to add) it will be fail.

Reply:
Could be also like Token or Password, it's not because it's open source that there is no security / authentification.

Reply:
I don't have a strong opinion about adding it, however, I'm doubtful that we'll add secrets that we have (mostly used in CI) to our code-base

Reply:
Removed with 8046a5f